lib/sufia/arkivo/create_subscription_job.rb

```diff
@@ -27,41 +27,41 @@ def run
 
       private
 
-      def logger
-        ActiveFedora::Base.logger
-      end
+        def logger
+          ActiveFedora::Base.logger
+        end
 
-      def validate_user!
-        log_and_raise('User not found') if user.blank?
-        log_and_raise('User does not have an Arkivo token') if user.arkivo_token.blank?
-        log_and_raise('User has not yet connected with Zotero') if user.zotero_userid.blank?
-        log_and_raise('User already has a subscription') if user.arkivo_subscription.present?
-      end
+        def validate_user!
+          log_and_raise('User not found') if user.blank?
+          log_and_raise('User does not have an Arkivo token') if user.arkivo_token.blank?
+          log_and_raise('User has not yet connected with Zotero') if user.zotero_userid.blank?
+          log_and_raise('User already has a subscription') if user.arkivo_subscription.present?
+        end
 
-      def log_and_raise(message)
-        logger.error "*** Arkivo error: #{message}: #{user}"
-        raise SubscriptionError.new(message)
-      end
+        def log_and_raise(message)
+          logger.error "*** Arkivo error: #{message}: #{user}"
+          raise SubscriptionError, message
```

Comment:
This is new to me but apparently it's the way to raise exceptions http://ruby-doc.org/core-2.1.1/Kernel.html#method-i-raise